CMakeLists.txt

```diff
@@ -65,11 +65,4 @@ target_link_libraries(${PROJECT_NAME} ${LIBRARY_LIST})
 
 target_link_libraries(
   ${PROJECT_NAME}
-  LLVMTransformUtils
-  LLVMAnalysis
-  LLVMTarget
-  LLVMOption # Support
-  LLVMObject # BitReader, Core, Support
-  LLVMBitReader # Core, Support
-  LLVMCore # Support
-  LLVMSupport)
+  LLVM)
```

Comment:
Thanks for trimming this -- IIRC we had to specify all of these manually for a much earlier version of LLVM.